src/NuGet.Clients/NuGet.PackageManagement.UI/Options/MappingUIDisplay.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.Options
+{
+    public class MappingUIDisplay
+    {
+        public string ID { get; set; }
```

Comment:
Is it just package Id or packagePattern? Maybe I rename to `PackageId` or `PackagePattern`, it's quite hard to remember it was packageId to begin.